### ISSUE TITLE
Fixing applications that are installed under /opt

### ISSUE THREAD

**Issue by temporaryMan1233** [Foggalong/hardcode-fixer#331]
Hello

Some applications like TeamViewer are installed under /opt (/opt/teamviewer for example).
Any way to make hardcode-fixer respect such locations?

Thank you
Temp Man

**Comment by Foggalong** [comment]
@saveriob Is this that the launcher is under opt also or is the launcher in one of the normal locations?

**Comment by palob** [comment]
The launcher has got to be elsewhere, otherwise it wouldn't be picked up.
AFAIK /opt isn't looked up for launchers.

**Comment by bilelmoussaoui** [comment]
Totally agree with @palob here. Desktop files should be installed under a 
```
$XDG_DATA_DIRS/applications
```
Like /usr/share, /usr/local/share or $home/.local/share.

**Comment by Foggalong** [comment]
@bilelmoussaoui @palob I agree also, I just meant was enquiring where it was in this instance :sweat_smile:
